packages/rpc-core/src/rpc-methods/__typetests__/get-block-type-test.ts

```diff
@@ -0,0 +1,1256 @@
+import { Base58EncodedAddress } from '@solana/addresses';
+import { Blockhash, TransactionVersion } from '@solana/transactions';
+import assert from 'assert';
+
+import { TransactionError } from '../../transaction-error';
+import {
+    Base58EncodedBytes,
+    Base58EncodedDataResponse,
+    Base64EncodedDataResponse,
+    LamportsUnsafeBeyond2Pow53Minus1,
+    Reward,
+    TokenBalance,
+    TransactionStatus,
+    U64UnsafeBeyond2Pow53Minus1,
+} from '../common';
+import { mockTypeTestResponse, mockTypeTestRpc } from './common';
+
+const rpc = mockTypeTestRpc();
```

Comment:
I worry that this makes people have to jump around between files to learn what's going on. What if you did this instead?
```suggestion
const rpc = null as unknown as Rpc<SolanaRpcMethods>;
```

Reply:
ah nuts